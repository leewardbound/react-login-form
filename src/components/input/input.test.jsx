import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Input from '.';

const renderer = new ShallowRenderer();

describe('Input', function () {
  it('should render correctly', function () {
    ['email', 'password'].forEach(type => {
      const component = renderer.render(
        <Input type={type} name={type} />
      );

      expect(component).toMatchSnapshot();
    });
  });
});
