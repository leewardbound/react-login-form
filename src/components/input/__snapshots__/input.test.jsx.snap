// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render correctly 1`] = `
<input
  className="jpmZcP"
  name="email"
  type="email"
/>
`;

exports[`Input should render correctly 2`] = `
<input
  className="jpmZcP"
  name="password"
  type="password"
/>
`;
